examples/nxt-screen.py

```diff
@@ -73,6 +86,7 @@ def main_loop():
     b.play_tone(440, 250)
 
     while True:
+        mouse = pygame.mouse.get_pos()
```

Comment:
You can use event.pos instead and get accurate mouse position at the time of the click.

Reply:
removed line 89 and used event.pos